test/test_Success.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+from aperturedb.ParallelQuery import ParallelQuery
+from aperturedb.Query import QueryBuilder
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class TestLoaderSuccess():
+    def assertEqual(self, expected, actual):
+        if expected != actual:
+            raise AssertionError(
+                "Expected {}, got {}".format(expected, actual))
+
+    def assertTrue(self, condition):
+        if not condition:
+            raise AssertionError("Condition not true")
```

Comment:
This would end up in a very obscure message in case of test failing.
pytest assertions have a very nice contextual info added automatically when asserts fail.
https://docs.pytest.org/en/7.1.x/how-to/assert.html

Reply:
should be fixed